packages/client/src/links/wsLink.ts

```diff
@@ -128,7 +128,13 @@ export function createWSClient(opts: WebSocketClientOptions) {
   }
 
   function createWS() {
-    const conn = new WebSocketImpl(url);
+    let urlString: string;
+    if (typeof url === 'function'){
+      urlString = url();
+    } else{
+      urlString = url;
+    }
```

Comment:
Use a ternary here instead of `if/else` please

Reply:
done